src/kepler_model/estimate/model/model.py

```diff
@@ -5,7 +5,15 @@
 
 from kepler_model.estimate.model.curvefit_model import CurveFitModelEstimator
 from kepler_model.estimate.model.scikit_model import ScikitModelEstimator
-from kepler_model.estimate.model.xgboost_model import XgboostModelEstimator
+
+have_xgboost = True
+try:
+    import xgboost
+except ImportError:
+    have_xgboost = False
```

Comment:
```suggestion
import importlib.util 

HAS_XGBOOST = importlib.util.find_spec("xgboost") is not None
```

also consider using python convention of naming constants in UPPERCASE

Reply:
any advantage of using importlib than  importing the package directly?

Reply:
yes, we don't want to import xgboost  (unused import)

Reply:
actually we want to keep the import for the case when we want to use the xgboost trainers. only change needed is to add back xgboost dependency in pyproject.toml

@KaiyiLiu1234 @sunya-ch please comment here.